api/prometheus/v1/api.go

```diff
@@ -404,6 +407,20 @@ type queryResult struct {
 	v model.Value
 }
 
+// TSDBResult contains the result from querying the tsdb endpoint
+type TSDBResult struct {
+	SeriesCountByMetricName     []MetricCountPair `json:"seriesCountByMetricName"`
+	LabelValueCountByLabelName  []MetricCountPair `json:"labelValueCountByLabelName"`
+	MemoryInBytesByLabelName    []MetricCountPair `json:"memoryInBytesByLabelName"`
+	SeriesCountByLabelValuePair []MetricCountPair `json:"seriesCountByLabelValuePair"`
+}
+
+// MetricCountPair models information about metric name and count
+type MetricCountPair struct {
```

Comment:
I would vote for renaming this to `Stat` (:

Reasons:

* It is called `stat` in Prometheus
* It's not technically a "metric"
* It's not always a "count" 

What do you think? :hugs:

Reply:
Makes sense to me. I was focusing on a single use case but this is better.